checks/caesar.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import File, Test, Error, check
```

Comment:
Sorry for missing this-- we could definitely simplify to `from check50 import *` without issue if that's easier.